app/forms/new_article_form.rb

```diff
@@ -0,0 +1,12 @@
+class NewArticlesForm
+    include ActiveModel::Model
+
+
+    attr_accessor(
+        :title,
+        :text
+    )
+
+    validates :title, presence: true
+    validates :text, presence: true
```

Comment:
When your attributes are validated with the same scope, you can group them together

```suggestion
    validates(
      :title,
      :text,
      presence: true
    )
```